crates/fluvio-cluster/src/delete.rs

```diff
@@ -131,23 +132,36 @@ impl ClusterUninstaller {
 
     async fn uninstall_local(&self) -> Result<(), ClusterError> {
         let pb = self.pb_factory.create()?;
-
         pb.set_message("Uninstalling fluvio local components");
-        Command::new("pkill")
-            .arg("-f")
-            .arg("fluvio cluster run")
-            .output()
-            .map_err(UninstallError::IoError)?;
-        Command::new("pkill")
-            .arg("-f")
-            .arg("fluvio run")
-            .output()
-            .map_err(UninstallError::IoError)?;
-        Command::new("pkill")
-            .arg("-f")
-            .arg("fluvio-run")
-            .output()
-            .map_err(UninstallError::IoError)?;
+
+        let kill_proc = |name: &str, commands: Option<&[String]>| {
```

Comment:
nit: I see that sysinfo api uses `cmd` but I think that maybe is more appropriate to use something like `command_args` instead of `commands`